src/Helpers.php

```diff
@@ -123,7 +123,7 @@ public static function wrapOCSResponse(Route $route, ControllerMethodResponse $r
 		return $schema;
 	}
 
-	public static function cleanEmptyResponseArray(array|stdClass $schema): array|stdClass {
+	public static function cleanEmptyResponseArray(array $schema): array|stdClass {
```

Comment:
👀

Reply:
When calling toArray on OpenApiType and stdClass can be returned. In some cases this method is directly called with the result so we need to allow it here.